src/App.js

```diff
@@ -47,7 +123,7 @@ function App() {
         <>
           <h1>Todo List</h1>
```

Comment:
If you try moving this `h1` above/outside of the `{isLoading ? }` ternary, you might find it feels like the app is loading "quicker" via having some parts of the page load in before the AJAX request is complete.